.prettierignore

```diff
@@ -1,3 +1,4 @@
 contracts/governance/GovernorAlpha.sol
 contracts/governance/Timelock.sol
 contracts/governance/VoteToken.sol
+contracts/true-currencies/tokens/TrueUSDNoDelegate.sol
```

Comment:
What's the .prettierignore reason? Can we fix it during this upgrade or would it be too much effort?

Reply:
Thanks for the explanation later on, makes sense. Removed `TrueUSDNoDelegate.sol` and references to it in the linters' ignore files.